Spring-Boot/Spring-Boot-Problem/src/main/java/tw/waterball/judgegirl/springboot/problem/repositories/MongoProblemRepository.java

```diff
@@ -111,6 +108,18 @@ public Problem save(Problem problem, Map<LanguageEnv, InputStream> providedCodes
         return mongoTemplate.save(problem);
     }
 
+    @Override
+    public int saveProblemWithTitleAndGetId(String title) {
+        Random random = new Random();
+        int id = random.nextInt();
+        Problem problem = Problem.builder()
+                .id(id)
+                .title(title)
+                .build();
+        mongoTemplate.insert(problem, "problem");
+        return id;
+    }
```

Comment:
We can't use Random to generate the next ID, there might be conflicts.
I think you can count up our id in this way:
```
problemCount = countTheProblemDocumentsFromMongo
int id = 70000 + problemCount
```

Reply:
btw,   you should use `mongoTemplate.save(problem)`, if you create the Problem instance directly